volmaster/daemon.go

```diff
@@ -95,7 +96,7 @@ func (d daemonConfig) handleRemove(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := removeVolume(vc); err != nil {
+	if err := removeVolume(vc, d.timeout); err != nil {
```

Comment:
this is minor but I am wondering if we should make `removeVolume()` and `createVolume()` `Daemon`'s methods. That way we don't need to pass daemon config around as args?